authentication.js

```diff
@@ -20,4 +36,4 @@ async function authentication(request, response, next) {
 };
 
 
-module.exports = authentication;
\ No newline at end of file
+module.exports = { validate, authentication };
```

Comment:
Great, almost there! Remember to rename the file to `middlewares.js` before merging